### DISCUSSION TITLE
Version 0.5.0 Beta 7 Released

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1371] [Announcements]
https://github.com/DSheirer/sdrtrunk/releases/tag/v0.5.0-beta.7

**Comment by JurisTeiv** [comment]
Hi, I have a problem with mp3 records, it sounds like speed is turned to very very slow level

**Comment by DSheirer** [comment]
Download the software again.  There was an mp3 issue with the first release.  I posted an updated release.  You must have downloaded it right away.

**Reply by JurisTeiv** [reply]
I've downloaded source code 5 hours ago and have a problem with mp3

**Reply by DSheirer** [reply]
@JurisTeiv this MP3 issue should be resolved in the final release (today).  Please let me know if you are still having issues.

**Reply by JurisTeiv** [reply]
Thank you @DSheirer everything is ok

**Comment by kernworks** [comment]
@DSheirer should probably untag this as an announcement.
It's showing up on top of the Discussions page still.
